ca/server.go

```diff
@@ -36,6 +36,10 @@ type Server struct {
 	joinTokens                  *api.JoinTokens
 	reconciliationRetryInterval time.Duration
 
+	// skipAutoUpdateRootCA, if set to true, causes the CA server to skip automatically updating
+	// the signing credentials based on changes in the memory store
+	skipAutoUpdateRootCA bool
```

Comment:
Ok, I don't feel super strongly about my position - it's more of a preference, so happy to address this.